news/2017-11-28-beatmap-spotlights-october-2017.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "Beatmap Spotlights October 2017"
+date: 2017-11-28 16:00:00 +0000
+---
+
+Halloween may be over, but with that comes a selection of the best maps from the spooky month of October. It’s time to look at this month’s Spotlights, and maybe get a medal while you’re at it!
```

Comment:
this pr -> `’`
my keyboard -> `'`
strange... 🤷‍♂️

Reply:
we should use standard `'` characters for the time being